src/components/Editor/FileExplorer/FilesList.tsx

```diff
@@ -135,7 +135,7 @@ const FilesList = ({ isExplorerCollapsed }: FileListProps) => {
         <MenuList
           title="Contracts"
           itemType={EntityType.ContractTemplate}
-          items={project.contractTemplates}
+          items={project.contractTemplates.sort(c => c.index)}
```

Comment:
to avoid mutation
```suggestion
          items={project.contractTemplates.slice().sort(c => c.index)}
```
or
```suggestion
          items={[...project.contractTemplates].sort(c => c.index)}
```

Reply:
I agree, but doing an experiment to see if this effect staging.